illumos does not support Windows Code Page 950, so defer to basic BIG5 instead.

--- cups-2.4.1/cups/language.c.orig	2022-08-21 08:59:22.932692369 +0000
+++ cups-2.4.1/cups/language.c	2022-08-21 09:01:05.855253912 +0000
@@ -69,7 +69,7 @@
 			  "unknown",		"unknown",
 			  "unknown",		"unknown",
 			  "cp932",		"cp936",
-			  "cp949",		"cp950",
+			  "cp949",		"big5",
 			  "cp1361",		"bg18030",
 			  "unknown",		"unknown",
 			  "unknown",		"unknown",
